core/templates/dev/head/pages/story_editor/story_editor.html

```diff
@@ -16,12 +16,50 @@
 
 {% block content %}
 <div ng-controller="StoryEditor">
+  <a class="btn btn-primary back-to-topic" ng-href="/topic_editor/<[topicId]>">
+    <i class="fa fa-arrow-left"></i> &nbsp;Go back to topic
+  </a>
   <story-editor></story-editor>
 </div>
+
+<style>
+  .back-to-topic {
```

Comment:
Scope this so that it is restricted to this page -- e.g. put a class in the parent div and prefix the CSS rule with it, or something similar.

Also maybe say something like .back-to-topic-button

Reply:
Done.